1. CWE Base Mod/Cold War Era Mod (CWE)/common/goods/00_goods.txt

```diff
@@ -868,7 +866,7 @@ refined_oil = {
 
 silk = {
 	texture = "gfx/interface/icons/goods_icons/silk.dds"
-	cost = 10
+	cost = 20
```

Comment:
Base prices should be kept uniform at 10 for raw resource, 20 for intermediate and 40 for completed products. This should not be an issue as actual prices will be determined by market forces anyway.